sc/social/like/utils.py

```diff
@@ -101,3 +103,12 @@ def _image_size(current, new):
         if n_width > width:
             return current
     return (width, height)
+
+
+def validate_canonical_domain(value):
+    """Check if the value is a URI containing only scheme and netloc."""
+    _ = urlparse(value)
```

Comment:
in fact the name of the variable doesn't matter here, we only take care of the attributes and `_` is shorter than `parsed`; see example 5 on [The Meaning of Underscores in Python](https://dbader.org/blog/meaning-of-underscores-in-python).